experimental/add_to_app/SimpleAndroidApp/app/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.simpleandroidapp"
+        minSdkVersion 19
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation 'android.arch.lifecycle:common-java8:1.1.1'
```

Comment:
Just an FYI, hopefully we can remove this requirement soon when @blasten is able to add POM support in the engine.

Reply:
This will be fixed by https://github.com/flutter/flutter/pull/39157

Reply:
Got it. I'll keep an eye out. These apps will need to mutate several times as y'all make progress.